userguide/platform/includes/killbill-configuration-properties-table.adoc

```diff
@@ -976,7 +976,7 @@ KB_org_killbill_security_skipAuthForPlugins   |Specifies whether authentication
 |===
 |Property Name/Environment Variable Name   |Description   |Default Value | Configuration Method
 //-------------------------------------------------
-|org.killbill.catalog.frequentValuesCacheSize   |Coming Soon   |1000   |System property
+|org.killbill.catalog.frequentValuesCacheSize   |Number of values to be considered for cache   |1000   |System property
```

Comment:
I believe this is used in the [DefaultPrice](https://github.com/killbill/killbill/blob/33aa3e9a65bf40de10edee7429341370488d4fb5/catalog/src/main/java/org/killbill/billing/catalog/DefaultPrice.java#L44) class. So, maybe specify that it is applicable for price and elaborate it slightly? The logic for this is [here](https://github.com/killbill/killbill/blob/33aa3e9a65bf40de10edee7429341370488d4fb5/catalog/src/main/java/org/killbill/billing/catalog/DefaultPrice.java#L88).